readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
Sorry, I misunderstood you :) yea, we can enforce some restrictions depending on whether it's a mixin, value, property, or selector in the runtime. We'd only have to introduce different function types here.